2.4/root/usr/bin/run-mongod

```diff
@@ -74,26 +74,23 @@ if [ -z "${MONGODB_REPLICA_NAME-}" ]; then
     usage
   fi
   # Run the MongoDB in 'standalone' mode
-  if [ ! -f ${MONGODB_DATADIR}/.mongodb_datadir_initialized ]; then
-    # Create MongoDB users and restart MongoDB with authentication enabled
-    # At this time the MongoDB does not accept the incoming connections.
-    mongod $mongo_common_args & #--bind_ip 127.0.0.1 --quiet >/dev/null &
-    wait_for_mongo_up
+  mongod $mongo_common_args &
+  wait_for_mongo_up
+  if [ $(mongo admin --quiet --eval "db.system.users.find({'user':'admin'}).length()") == "1" ]; then
```

Comment:
>What is a difference count() vs length()?

In case of using `collection.find().length()` you're fetching results from MongoDB and after that use `length()` for counting how many rows were returned. You're ignoring results completely and don't use it. It's the same as using `SELECT COUNT(*)` vs `SELECT *` and counting rows manually.

I'm suggesting explicitly ask mongodb to count rows for us and return result. For a huge amount of data it could be faster (but it's not our case) and also it's more clean way IMHO.